google/cloud/bigtable/examples/data_async_snippets.cc

```diff
@@ -166,6 +166,42 @@ void AsyncCheckAndMutate(cbt::Table table, cbt::CompletionQueue cq,
   //! [async check and mutate]
   (std::move(table), std::move(cq), argv[1]);
 }
+
+void AsyncReadModifyWrite(cbt::Table table, cbt::CompletionQueue cq,
+                          std::vector<std::string> argv) {
+  if (argv.size() != 2U) {
+    throw Usage{
+        "async-read-modify-write: <project-id> <instance-id> <table-id>"};
+  }
+
+  //! [async read modify write]
```

Comment:
Please add a `// TODO(#2404) - remove hard-coded key values` just before this. I just created that bug btw, no way for you to know about it.